script.js

```diff
@@ -1,6 +1,9 @@
-const ADD_BUTTON = document.getElementById('btnAdd');
 let tableArray = [];
-ADD_BUTTON.disabled = true;
+
+const isAddButtonDisabled = (isDisabled) =>
```

Comment:
funções que começam com `is` indicam que o retorno é um booleano pq vc esta fazendo uma pergunta. `setCreateButtonState`?

Reply:
Boa, vou renomear aqui.